properties doc

sgoomar opened the pull request:
Docs for frontier orbitals and excited states

OmerOmar committed:
properties doc

Docs for frontier orbitals and excited states

tneumann2021 reviewed (changes requested):
I suggest to adapt the titles of the files so users can directly associate them with the dropdown on the platform, e.g. "S, T, f: Excited state properties", and "HOMO, LUMO" instead of Frontier Orbitals. Otherwise users need to know what a property is before they can find it in our documentation. Let me know if this does not make sense to you, then we should discuss this.

OmerOmar committed:
Filenames changed

Filenames changed:
excitation --> S, T, f - Excited state properties
frontier --> HOMO, LUMO

OmerOmar committed:
Titles in docs changed

Titles changed so users can more easily relate property to documentation

OmerOmar committed:
Filenames reverted

OmerOmar committed:
index.rst updated to correct filenames

Commit:
fixed underline length
